packages/frontend-2/assets/css/tailwind.css

```diff
@@ -51,15 +52,18 @@
   .caption {
     @apply text-xs;
   }
-}
 
-/**
- * Ensuring ring offset color always defaults to bg color
- */
-*,
-[type='checkbox']:focus,
-[type='radio']:focus {
-  @apply ring-offset-background;
+  .default-width {
+    @apply mx-auto max-w-7xl px-4 sm:px-6 lg:px-8;
+  }
+
+  .background-special {
+    @apply bg-[radial-gradient(ellipse_at_top_left,_var(--tw-gradient-stops))] from-background-primary to-primary dark:bg-blue-700 dark:from-sky-400 dark:via-blue-700 dark:to-indigo-900;
```

Comment:
This feels like a class that you're going to add once in maybe app.vue or something, so I'm not sure if it has to be put inside `tailwind.css` as a globally re-usable class.

Reply:
unused, comment it out. if we reintroduce it, name it better (e.g. background-panel-gradient)